pints/toy/_eight_schools.py

```diff
@@ -0,0 +1,153 @@
+#
+# Eight schools log-pdf.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import pints
+
+from . import ToyLogPDF
+
+
+class EightSchoolsLogPDF(ToyLogPDF):
+    r"""
+    The classic Eight Schools example from [1]_. This model was used to
```

Comment:
This makes me feel like I'm supposed to know about this example, which is not what we expect from pints' target audience?

Reply:
Good point. Have changed tack a bit with the wording now. See what you think.